### DISCUSSION TITLE
Insufficient Emoji Reaction Options

### DISCUSSION THREAD

**Discussion by ashGlaw** [community/community#63021] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I believe the user experience, for desktop and mobile alike, would be improved by the addition of more choices of emoji to react to PRs and replies with. The eight available are able to express only a rather limited range of feelings, and it seems few people even agree on the meaning of some of them. Looking at you, sideeye emoji. If the meanings are already up to interpretation, seems like having more options would hardly add to the confusion, and could provide more clear options.

**Comment by anina-hitt** [comment]
+1

**Comment by benjub** [comment]
Related to #69934 and #10240.
